hooks/useRoomActions.ts

```diff
@@ -103,9 +103,16 @@ export default function useRoomActions() {
 
   function onEndTurn() {
     runTransaction(roomRef, (room: Room) => {
+      var newStatus: RoomStatus;
+      if (room.settings.maxRounds == room.round && room.currentTeamIndex == room.teams.length - 1){
```

Comment:
Let's not use `var`. Either it's let or const.
https://medium.com/@codingsam/awesome-javascript-no-more-var-working-title-999428999994